src/components/integrations/notion.tsx

```diff
@@ -48,26 +72,23 @@ function ParentItemDropdown({
       position={'bottom'}
       triggerId={triggerId}
       leftClick={true}
+      disabled={disabled}
     >
-      {
-        // @ts-ignore
-        ({ isOpen }) => (
-          <Button id={triggerId} size={'medium'} variant={'tertiary'}>
-            {selectedItem?.type === 'database' ? (
-              <Grid6 variant="Bold" size={16} />
-            ) : (
-              <Document variant="Bold" size={16} />
-            )}
-            {selectedItem ? (
-              <Text value={getDecorations(selectedItem.title)} block={emptyTextBlock} />
-            ) : (
-              'Error Loading Items'
-            )}
-            {isOpen && <ArrowDown2 size={16} />}
-            {!isOpen && <ArrowUp2 size={16} />}
-          </Button>
-        )
-      }
+      {({ isOpen }: { isOpen: boolean }) => (
```

Comment:
according to the Vercel logs this is failing to build -- perhaps you need the `// @ts-ignore` -- if you see the Vercel check fail in the PR, you can test it more easily locally by running `nom run build` that has to pass before Vercel can deploy it